tests/chocolatey-tests/choco-template.Tests.ps1

```diff
@@ -163,5 +191,81 @@ Describe "choco <_>" -ForEach @(
+            $Output.Lines | Should -Contain "2 Custom templates found at $env:ChocolateyInstall\templates"
+        }
+    }
+
+    Context 'Running <_> with verbose argument' {
+        BeforeAll {
+            Restore-ChocolateyInstallSnapshot
+
+            $null = Invoke-Choco install msi.template zip.template
+
+            $Output = Invoke-Choco $_ --verbose
+        }
+
+        It "Exits with Success (0)" {
+            $Output.ExitCode | Should -Be 0
+        }
+
+        It "Displays template information for msi" {
+            $Output.Lines | Should -Contain "Template name: msi"
+            $Output.Lines | Should -Contain "Version: 1.0.2"
+            $Output.Lines | Should -Contain "Default template: False"
+            $Output.Lines | Should -Contain "Summary: MSI Chocolatey template"
+            $Output.Lines | Should -Contain "### Chocolatey MSI template"
+            $Output.Lines | Should -Contain "This adds a template for MSI packages."
+            $Output.Lines | Should -Contain "List of files:"
+            $Output.Lines | Should -Contain "$env:ChocolateyInstall\templates\msi\msi.nuspec"
+            $Output.Lines | Should -Contain "$env:ChocolateyInstall\templates\msi\ReadMe.md"
+            $Output.Lines | Should -Contain "$env:ChocolateyInstall\templates\msi\tools\chocolateybeforemodify.ps1"
+            $Output.Lines | Should -Contain "$env:ChocolateyInstall\templates\msi\tools\chocolateyinstall.ps1"
+            $Output.Lines | Should -Contain "$env:ChocolateyInstall\templates\msi\tools\chocolateyuninstall.ps1"
+            $Output.Lines | Should -Contain "$env:ChocolateyInstall\templates\msi\tools\LICENSE.txt"
+            $Output.Lines | Should -Contain "$env:ChocolateyInstall\templates\msi\tools\VERIFICATION.txt"
+        }
+
+        It "Displays template information" {
+            $Output.Lines | Should -Contain "Template name: zip"
+            $Output.Lines | Should -Contain "Version: 1.0.0"
+            $Output.Lines | Should -Contain "Default template: False"
+            $Output.Lines | Should -Contain "Summary: Zip Chocolatey template"
+            $Output.Lines | Should -Contain "### Chocolatey Zip template"
+            $Output.Lines | Should -Contain "This adds a template for archive (zipped) packages."
+            $Output.Lines | Should -Contain "List of files:"
+            $Output.Lines | Should -Contain "$env:ChocolateyInstall\templates\zip\zip.nuspec"
+            $Output.Lines | Should -Contain "$env:ChocolateyInstall\templates\zip\ReadMe.md"
+            $Output.Lines | Should -Contain "$env:ChocolateyInstall\templates\zip\tools\chocolateybeforemodify.ps1"
+            $Output.Lines | Should -Contain "$env:ChocolateyInstall\templates\zip\tools\chocolateyinstall.ps1"
+            $Output.Lines | Should -Contain "$env:ChocolateyInstall\templates\zip\tools\LICENSE.txt"
+            $Output.Lines | Should -Contain "$env:ChocolateyInstall\templates\zip\tools\VERIFICATION.txt"
+        }

```

Comment:
These two it blocks are very close to the same. I wonder if we could streamline this with a single it block and a foreach inside 🤔 

Perhaps something like:

```powershell
It "Displays <Name> template information" -ForEach @(
    @{ Name = 'msi' ; Lines = @( <Insert lines here> )
    @{ Name = 'zip' ; Lines = @( <Insert lines here> )
) {
    Foreach ( $expectedLine in $Lines ) {
        $Output.Lines | Should -Contain $expectedLine
    }
}
```